### PR TITLE
Create petr-troyanskii.md

### PR DESCRIPTION
**Pull request by ovesyan**

### CODE REVIEW

File: history/petr-troyanskii.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+description: Scholar who filed a patent for a mechanical translator in 1933
+---
+# Petr Troyanskii
+Petr Troyanskii was an educator and scholar who came up with an automatic bilingual dictionary in 1933. 
+
```
Comment (bittlingmayer) [inline history/petr-troyanskii.md:6]: > In 1935, he filed and received USSR Patent 40995 for his invention.
>
> > ***Машина для подбора и печатания слов при переводе с одного языка на другой или на нескольких языках одновременно***
>> *Machine for selecting and typing words when translating from one language to another or across multiple languages simultaneously.*

File: history/petr-troyanskii.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+The method suggested coding and interpreting grammatical functions using universal symbols. 
+The interlingual grammatical rules were based on Esperanto.
+
```
Comment (bittlingmayer) [inline history/petr-troyanskii.md:9]: > In the same years, [Georges Artsrouni](/people/georges-artsrouni.md) independently filed a patent in Paris.

File: history/petr-troyanskii.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+Troyanskii entered the University of St. Petersburg but did not finish it due to World War I. 
+Later he studied at the Institute of Red Professors. 
+
+He taught social sciences and the history of science and technology.
```
Comment (bittlingmayer) [inline history/petr-troyanskii.md:13]: mechanizing with z if we're going to be consistent

File: history/petr-troyanskii.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
+description: Scholar who filed a patent for a mechanical translator in 1933
+---
+# Petr Troyanskii
+Petr Troyanskii was an educator and scholar who came up with an automatic bilingual dictionary in 1933. 
+
+The method suggested coding and interpreting grammatical functions using universal symbols. 
+The interlingual grammatical rules were based on Esperanto.
+
+Troyanskii entered the University of St. Petersburg but did not finish it due to World War I.
```
Comment (bittlingmayer) [inline history/petr-troyanskii.md:10]: > ### Career

File: history/petr-troyanskii.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+---
+description: Scholar who filed a patent for a mechanical translator in 1933
```
Comment (bittlingmayer) [inline history/petr-troyanskii.md:2]: > Early machine translation scholar
?

File: history/petr-troyanskii.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+---
+# Petr Troyanskii
+Petr Troyanskii was an educator and scholar. In 1935, he filed and received USSR Patent 40995 for his invention. 
+
+The machine selected and typed words when translating from one language to another 
+or across multiple languages simultaneously.
+
+The method suggested coding and interpreting grammatical functions using universal symbols. 
+The interlingual grammatical rules were based on Esperanto. In the same years, 
+[Georges Artsrouni] (georges-artsrouni.md) independently filed a patent in Paris.
```
Comment (bittlingmayer) [inline history/petr-troyanskii.md]: No space, and use the full path:

[Georges Artsrouni](/people/georges-artsrouni.md)

File: history/petr-troyanskii.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+The method suggested coding and interpreting grammatical functions using universal symbols. 
+The interlingual grammatical rules were based on Esperanto. In the same years, 
+[Georges Artsrouni] (history/georges-artsrouni.md) independently filed a patent in Paris. 
+
+Troyanskii entered the University of St. Petersburg but did not finish it due to World War I. 
+Later he studied at the Institute of Red Professors. 
+
+He taught social sciences and the history of science and technology. 
+He considered mechanizing translation the cause of his whole life.
```
Comment (bittlingmayer) [inline history/petr-troyanskii.md:18]: This is somewhat subjective.  If there's a quote from him or something, that would be more factual.

Comment (ovesyan) [reply history/petr-troyanskii.md:18]: I read a book chapter by Hutchins and Lovtskii "Petr Petrovich Troyanskii (1894–1950): A Forgotten Pioneer of Mechanical Translation", and it resonated with what was written in Wikipedia. I will try to reformulate this based on more factual statements.

File: history/petr-troyanskii.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+The machine selected and typed words when translating from one language to another 
+or across multiple languages simultaneously.
+
+The method suggested coding and interpreting grammatical functions using universal symbols. 
+The interlingual grammatical rules were based on Esperanto. In the same years, 
+[Georges Artsrouni](/people/georges-artsrouni.md) independently filed a patent in Paris. 
+
+Troyanskii dedicated almost two decades of his life to the mechanization of translation. He put forward such ideas as:
+* pre- and post-editing of the text
```
Comment (bittlingmayer) [inline history/petr-troyanskii.md:15]: Could link post-editing to the post-editing article

File: history/petr-troyanskii.md (hunk 7 above)
Comment (bittlingmayer) [reply history/petr-troyanskii.md]: Thanks!

Comment (bittlingmayer) [reply history/petr-troyanskii.md]: In that case, the best way to do it is to just put a direct quote from that book.

**Pull request merged**